<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookStore</class>
 <widget class="QWidget" name="BookStore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BookStore</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="startWindow">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>941</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="3" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>books</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="deliveryTo">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>days</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Initial storage (50 - 1000)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="period">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Request delivery time from (1 - 4)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>                 to (2 - 5)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>days</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="initStore">
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Modeling period (10 - 30)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="deliveryFrom">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>                 to (2 - 4)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="ordNumFrom">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Number of orders from (1 - 3)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QSpinBox" name="ordNumTo">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>orders</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Books amount in order from (1 - 4)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="ordBooksFrom">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>                 to (2 - 5)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QSpinBox" name="ordBooksTo">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>books</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QSpinBox" name="margin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Margin (1 - 100)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>455</y>
         <width>951</width>
         <height>28</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="exitButton_1">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="setupButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 85, 0);</string>
          </property>
          <property name="text">
           <string>Setup</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="storeWindow">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="1">
          <widget class="QTableView" name="requests"/>
         </item>
         <item row="2" column="1">
          <widget class="QTableView" name="store"/>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QTableView" name="orders"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>ORDERS</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>STORAGE</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>REQUESTS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="exitButton_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 0, 0);</string>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="2">
            <widget class="QLabel" name="labelProc">
             <property name="text">
              <string>Process waits to start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="endDay">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="currentDay">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>of</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="startButton">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 85, 0);</string>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>days</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="reportButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 85, 0);</string>
             </property>
             <property name="text">
              <string>Show report</string>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
